homeassistant/components/flux/switch.py

```diff
@@ -320,6 +326,13 @@ def is_on(self):
                 time_state,
                 now,
             )
+            self._attributes = {}
+            self._attributes["Colour Temperature (K)"] = int(temp)
```

Comment:
All attributes should be in snake_case see https://www.python.org/dev/peps/pep-0008/

Reply:
changed _attributes to _state_attributes in 25a8a0206500d22981d8297fceaaf9038341609b